pom.xml

```diff
@@ -76,6 +76,8 @@
 		<mockito.version>4.6.1</mockito.version>
 		<jetbrains.annotations.version>23.0.0</jetbrains.annotations.version>
 		<log4j2.version>2.17.2</log4j2.version>
+		<testcontainer.version>1.17.3</testcontainer.version>
+		<urlfrontier.version>2.2</urlfrontier.version>
```

Comment:
having module specific variables in the top most pom defeats the objects of putting things into modules

Reply:
yes, that would be perfect